pkg/workload/core.go

```diff
@@ -456,17 +452,10 @@ func (c *core) doTransactionReadModifyWrite(ctx context.Context, db ycsb.DB, sta
 	}
 	defer c.putValues(values)
 
-	readValues, err := db.Read(ctx, c.table, keyName, fields)
-	if err != nil {
-		return err
-	}
-
-	if err := db.Update(ctx, c.table, keyName, values); err != nil {
-		return err
-	}
-
-	if c.dataIntegrity {
-		c.verifyRow(state, keyName, readValues)
+	if t, ok := db.(*client.DbWrapper); ok {
```

Comment:
very wired that the core knows client module.

Reply:
I agree. But if not, we need to add ReadModifyWrite to DB interface and every DB need to implement that.

Reply:
please check how BatchDB does